SmartDeviceLink/private/SDLMenuManager.m

```diff
@@ -530,6 +529,33 @@ - (BOOL)sdl_isDynamicMenuUpdateActive:(SDLDynamicMenuUpdatesMode)dynamicMenuUpda
     }
 }
 
+/// Checks if 2 or more cells have the same text/title. In case this condition is true, this function will handle the presented issue by adding "(count)".
+/// E.g. Choices param contains 2 cells with text/title "Address" will be handled by updating the uniqueText/uniqueTitle of the second cell to "Address (2)".
+/// @param choices The choices to be uploaded.
+- (void)sdl_addUniqueNamesToCells:(nullable NSArray<SDLMenuCell *> *)choices {
```

Comment:
The cells are displaying on the head unit as:

```
Item (3)
Item (2)
Item
```

It should be:

```
Item
Item (1)
Item (2)
```

---

When testing with a subcell item (4th item) it came out as:

```
Item (3)
Item (2)
Item
Item (4) - Had subcells, but they would not appear